content/connect/index.textile

```diff
@@ -0,0 +1,263 @@
+---
+title: Connections
+meta_description: "Establish and maintain a persistent connection to Ably using the Realtime SDK."
+languages:
+  - javascript
+  - nodejs
+  - ruby
+  - java
+  - swift
+  - objc
+  - csharp
+  - flutter
+---
+
+p(note). This section applies to the Ably Realtime Client only.
+
+The Ably Realtime SDK establishes and maintains a connection to the Ably service, using the most efficient transport available, typically "WebSockets":https://ably.com/topic/websockets. The Ably realtime protocol operates and multiplexes all channel traffic over that connection. Channels are separate from Connections and are used to logically separate messages into different topics, see the "channels documentation":/channels for further information.
+
+h2(#getting-started). Creating a connection
+
+The Ably Realtime SDK opens and maintains a connection to the Ably realtime servers on instantiation. The @Connection@ object provides a straightforward API to monitor and manage "connection state":/connect/states.
```

Comment:
I think part of this content should form the beginnings of the caveat mentioned about instantiation and connecting.